core/src/main/java/fr/sncf/osrd/standalone_sim/ScheduleMetadataExtractor.kt

```diff
@@ -100,9 +100,11 @@ fun run(
 
     // Compute stops
     val stops = ArrayList<ResultStops>()
+    var cumulateStopTime = 0.0
     for (stop in schedule.stops) {
         val stopTime = ResultPosition.interpolateTime(stop.position, headPositions)
-        stops.add(ResultStops(stopTime, stop.position, stop.duration))
+        stops.add(ResultStops(stopTime + cumulateStopTime, stop.position, stop.duration))
+        cumulateStopTime += stop.duration
```

Comment:
I'm a little confused.

We already have built-in methods to consider the stop time, the proper way to get the time would be `envelopeWithStops.interpolateTotalTime(stop.position)`. It (should) include the stop time when getting the exact stop position (though I'm not 100% confident that it handles epsilon diff well). 

I'm surprised that `ResultPosition.interpolateTime` even exists. Its docstring indicates that it differs with `envelope.interpolateTime` in that it *does* consider stop time.